### DISCUSSION TITLE
Gore post detection

### DISCUSSION THREAD

**Discussion by atomic-junky** [Docile-Alligator/Infinity-For-Reddit#889] [Ideas]
I think it would be a good idea if there was a setting to blur/hide the gore images.

Posts can be checked by making a request to an API or by using a library (I don't know if it exists).

What do you think about this idea?

I can try to do it but I'm not sure if this is the best way to do it...

**Comment by powerjungle** [comment]
You mean separating NSFW/NSFL/Gore and bluring them individually? Will this work with flairs or something?

**Reply by atomic-junky** [reply]
Yes

**Comment by Sayasam42** [comment]
It's a good idea. The app could automatically blur posts that have either one of these tags:
- NSFL
- Not Safe For Life
- NSFL (Not Safe For Life)
